app/views/articles/index.html.erb

```diff
@@ -13,10 +13,10 @@
         <td><%= link_to article.title, article %></td>
         <td><%= link_to article.body, article %></td>
         <% article.images.each do |image|%>
-          <td><%= image_tag(article.images[image]) %></td>
+          <td><%= image_tag image, height:"100px", width:"100px" %></td>
```

Comment:
Please always use `()` for a function. i.e `image_tag(image, height:"100px", width:"100px")`

Reply:
Updated